README.md

```diff
@@ -94,11 +94,17 @@
 1. Run `./hack/delete.sh $RESOURCEGROUP` to delete the deployed cluster.
 
 ##### Access the cluster
- Cluster can be accessed via `UI` and `CLI`. If cluster was created using AAD
-integration (Pre-requisites 7.4), you can login using Azure AD. Second option, 
-which will be deprecated later, is `htpasswd`. Username for it is `osadmin`, 
-password - random generated. To get random generated password execute: 
-`./hack/config.sh get-config $RESOURCEGROUP | jq -r .config.AdminPasswd`
+A cluster can be accessed via the `UI` or `CLI`. If it was created using AAD
+integration (Pre-requisites 7.4), you can login using Azure AD. Another option,
```

Comment:
@mjudeikis what's 7.4?

Reply:
Section 7. Point 4. Just when you watch it in MD where everything is `1 1 1 1 1` its not nice to find it